src/app/modules/verification/verification.errors.ts

```diff
@@ -82,3 +82,14 @@ export class NonVerifiedFieldTypeError extends ApplicationError {
     )
   }
 }
+
+/**
+ * Agency user has sent too many SMSes using default Twilio credentials
+ */
+export class SmsLimitExceededError extends ApplicationError {
+  constructor(
+    message = 'You have exceeded the free sms limit. Please refresh and try again.',
```

Comment:
This message will be shown to the user right? I don't think refreshing the page would change anything if the sms limit is reached. Maybe consult with @syan-syan

Reply:
will clarify with sonjia wrt this! what changes on page refresh is that now sms toggling would be disabled tgt with link shown to address billing

Reply:
confirmed that this is ok; will leave it as is